build.gradle

```diff
@@ -3,17 +3,20 @@
 buildscript {
   ext.kotlin_version = '1.4.20'
   ext.fragment_version = '1.2.0-rc01'
+  ext.jacocoVersion = '0.8.5'
   repositories {
     google()
     jcenter()
     gradlePluginPortal()
+    mavenCentral()
```

Comment:
I think we inadvertently add this in domain/build.gradle. Since you're adding this here, can you remove it from that Gradle file? It's a nice opportunistic fix.

Reply:
Done